stockpile/roles/dmidecode/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+# ensuring pip is installed
+- include_role:
+    name: install-pip
+
+# Checking if dmide is present
+- name: Verify Dmidecode
+  package:
+    name: dmidecode
+    state: present
+
+- name: Copy the script
```

Comment:
Can you use script module instead, it will save you from copying script and then executing it on remote?